README.md

```diff
@@ -263,6 +263,21 @@ local rx = radix.new({
 })
 ```
 
+### Parameters in Path with a custom regex
+
+You can specify parameters on a path and use a regex to match the parameter segment of the path (see `hsegment` of [RFC1738](https://www.rfc-editor.org/rfc/rfc1738.txt).
```

Comment:
why we need this feature? Are there any other product that supports similar feature?

I took a look at https://www.rfc-editor.org/rfc/rfc1738.txt, I do not sure if we need to support `Parameters in Path with a custom regex`.

Reply:
The reference is referring to [radixtree.lua Line 655](https://github.com/api7/lua-resty-radixtree/blob/master/lib/resty/radixtree.lua#L655) and that the custom regex is for the segment of the uri (aka everything between a `/`).

In regards to having this type of validation/feature I know of [prefix_regex in ambassador](https://www.getambassador.io/docs/edge-stack/latest/topics/using/prefix-regex), there is also [envoy route regex](https://www.envoyproxy.io/docs/envoy/v1.5.0/api-v1/route_config/route#route). 

However, I think we could also use the `var.request_uri` in `vars` to get the feature. This in turn means having a regex for 1) matching the route 2) matching the route with the argument, from my perspective this go's against the performance that APISIX seems to talk about then again I maybe wrong.

Reply:
This is a new feature for [lua-resty-radixtree](https://github.com/api7/lua-resty-radixtree), I need more time to research and think. I'll reply you before Friday in this week

If you were hurry about this new feature, feel free to ping me ^_^